<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserimentoFumetto</class>
 <widget class="QWidget" name="InserimentoFumetto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserimento Fumetto</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_barcode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_categoria">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distributore:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_distributore">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Editore:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_editore">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Collana:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_collana">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sottocollana:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_sottocollana">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prezzo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_prezzo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantit√†:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_quantita">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inserisci">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_titolo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titolo:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
